python/graphstorm/config/argument.py

```diff
@@ -748,19 +752,72 @@ def eval_fanout(self):
             # By default use -1 as full neighbor
             return [-1] * self.num_layers
 
+    @property
+    def textual_data_path(self):
+        """ distillation textual data path
+        """
+        if hasattr(self, "_textual_data_path"):
+            return self._textual_data_path
+        return None
+
+    @property
+    def lm_name(self):
+        """ HuggingFace Transformer-based model name for distillation
+        """
+        if hasattr(self, "_lm_name"):
+            return self._lm_name
+        return None
+
+    @property
+    def pre_trained_name(self):
+        """ Name of pre-trained HuggingFace model weights for distillation
+        """
+        if hasattr(self, "_pre_trained_name"):
+            return self._pre_trained_name
+        return None
+
+    @property
+    def max_global_step(self):
+        """ Maximum global size for distillation.
+        """
+        # only needed by distillation
+        if "distill" not in self.configuration["gsf"]:
+            return None
+        if hasattr(self, "_max_global_step"):
+            return self._max_global_step
+        else:
+            # default max global steps
+            return 10000
+
+    @property
+    def max_seq_len(self):
```

Comment:
Is this defined by the LM model?

Reply:
yes, this is the maximum sequence length for distillation.

Reply:
Can we move it into distill_lm_configs?

Reply:
You also need to add max_seq_len into [docs/source/configuration/configuration-run.rst](https://github.com/awslabs/graphstorm/pull/443/files#diff-975b3c593103b195865b6999a7230d36fa30ad20f661fc89ad4da43af28e7fb7)

Reply:
updated.